README.md

```diff
@@ -26,8 +26,9 @@ jobs:
       - uses: actions/checkout@v2
       - name: 'Bundler Audit'
         uses: andrewmcodes/bundler-audit-action@main
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          options: '--ignore CVE-2015-9284'
```

Comment:
excellent! Thank you! Hopefully it makes sense why I preferred this name but I would be happy to elaborate if not!